ui/packages/app/src/components/Dataset/ExampleComponents/Dataframe.svelte

```diff
@@ -43,6 +43,10 @@
 				} catch (e) {
 					console.error(e);
 				}
+			})
+			.catch((e) => {
+				loaded_value = value;
+				loaded = true;
```

Comment:
The `Dataframe` example component tries to fetch CSV files from the frontend, which is problematic for files in private spaces. This really should have been implemented in the backend, but in order to minimize too many changes in this PR, I added a quick workaround which is to just show the name of the file if it cannot be fetched.